src/std/ide/swank/completions.ss

```diff
@@ -0,0 +1,62 @@
+(import
+  :gerbil/expander
+  :std/sugar
+  :std/srfi/13
+  :std/iter
+  :std/srfi/1
```

Comment:
can you at least group the things properly? We really need a formatter just for you...